user/plan-sharing.md

```diff
@@ -0,0 +1,81 @@
+
+### Concurrency-based Plans
+Concurrency-based plans are available to all Travis CI accounts that use the Shared Plan, including Donor and Receiver accounts. When sharing a concurrency-based plan, the number of jobs is shared between all accounts. For example, if user account ***A*** decides to share a plan with 20 concurrent jobs with user accounts ***B*** and ***C***. This means that all three accounts have a total of 20 concurrent jobs. 
+
+Here is an example:
+User account ***A*** runs a build with 10 concurrent jobs. This consumes 10 of the 20 available slots. Then, Account ***B*** runs a build with 7 concurrent jobs, which consumes 17 out of the 20 available slots. Finally, if account ***C*** runs a build with 5 or more concurrent jobs, only 3 jobs are available to start, while the remaining jobs must wait in the queue for free slots.  
+
+
+### User limits
+If the donor’s plan has user license limits, only one pool of user licenses is consumed for both Donors and Receivers within the month. 
+
+A unique user triggering builds is checked against all accounts, not just one. 
+
+The user with the Donor account can see the number of unique users and which accounts consumed user licenses in a given month. The allowance gets deducted from the Donor’s allowance, and only the Donor account controls the user license allowance. 
+
+### License charges
+Users from ***Donor*** and ***Receiver*** accounts must consume a user license and incur the respective charges if applicable. 
+
+There is no double charge for the license for the same user, given that this user triggers a build and is a member of both the ***Donor*** and ***Receiver*** accounts simultaneously. 
+
+In other words, the user license consumption, which is a unique user triggering a build within a billing cycle, is tracked in the context of the Plan. Regardless of whether the source build is triggered from the ***Donor*** or ***Receiver*** accounts.
+
+### Credits
+The ***Donor*** account is responsible for the plan’s credits. 
+When the donor plan has credits, both the donor and the Receiver can use the credits from the account. Once the credits run out, the user with the donor account is responsible for refilling the credits. 
+
+The user with the Donor account can see the credit consumption information for the shared plan. This way, the donor user can identify which accounts consumed the credits. 
+
+### Trial Plans
+Trial Plans are not available for sharing. 
+
+A Travis CI Organization account (linked to an Assembla Space), which has been a RECEIVER of a shared Plan at least once, if such user selects a Travis CI Plan on its own, the user must not be able to see or enable a trial period or select a trial plan (unless there’s a higher than one trial period limit configured for a selected plan).
```

Comment:
(linked to an Assembla Space) - remove, the lead should state it and the description should be generic
RECEIVER or Receiver? Please keep it coherent in public docs